### PR TITLE
Add watch mode to client tests

### PR DESCRIPTION
**Pull request by nicksellen**
Add client test watching to match server test watching.

### CODE REVIEW

**Comment by simison** [comment]
Good one!

Could you add this to [package.json](https://github.com/Trustroots/trustroots/blob/master/package.json#L25-L27) as well?

These, too:
- https://github.com/Trustroots/trustroots/blob/master/INSTALL.md#running-tests
- https://github.com/Trustroots/trustroots/blob/master/INSTALL-DOCKER.md#running-tests

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "fontello": "gulp fontello",
     "test": "gulp test",
     "test:client": "gulp test:client",
+    "test:client:watch": "gulp test:client:watch",
```
Comment (nicksellen) [inline package.json]: I did already :)

Comment (simison) [inline package.json]: Oh, clearly bed time for me. :-)

**Comment by nicksellen** [comment]
Just added it to the install docs too now

**Comment by simison** [comment]
Tested, works great.

**Pull request merged**